cfme/tests/test_replication.py

```diff
@@ -1,10 +1,34 @@
 import pytest
 
 from cfme import test_requirements
+from cfme.utils.conf import credentials
+
 
 pytestmark = [test_requirements.replication]
 
 
+def setup_replication(remote_app, global_app):
+    """Configure global_app database with region number 99 and subscribe to remote_app."""
+    app_creds = {
+        "username": credentials["database"]["username"],
+        "password": credentials["database"]["password"],
+        "sshlogin": credentials["ssh"]["username"],
+        "sshpass": credentials["ssh"]["password"],
+    }
+    app_params = dict(region=99, dbhostname='localhost', username=app_creds['username'],
+                      password=app_creds['password'], dbname='vmdb_production',
+                      dbdisk=global_app.unpartitioned_disks[0], fetch_key=remote_app.hostname,
+                      sshlogin=app_creds['sshlogin'], sshpass=app_creds['sshpass'])
```

Comment:
Please create single `dict` instate of two as we are not using it multiple time. like
```
app_params = dict(
    region=99,
    dbhostname="localhost",
    username=credentials["database"]["username"],
    password=credentials["database"]["password"],
    dbname="vmdb_production",
    dbdisk=global_app.unpartitioned_disks[0],
    fetch_key=remote_app.hostname,
    sshlogin=credentials["ssh"]["username"],
    sshpass=credentials["ssh"]["password"],
)
```

Reply:
Good call @digitronik -- I agree with this as well